node/core/pvf/src/host.rs

```diff
@@ -851,15 +892,83 @@ fn pulse_every(interval: std::time::Duration) -> impl futures::Stream<Item = ()>
 	.map(|_| ())
 }
 
+// TODO: Should we purge unneeded binaries?
+// TODO: Test on windows.
```

Comment:
We're using POSIX sockets API for node-worker communication, windows is long not supported